### DISCUSSION TITLE
LF APAC开源布道者分享系列之一：如何有效规划时间 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#27] [Announcements]
# LF APAC开源布道者分享系列之一：如何有效规划时间 - 开源之道

https://opensourceway.community/posts/lfapac-evangelist-sharing/how-to-steal-time-from-your-boss/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

有关 摸鱼? 俺有曰过:

- [科学'摸鱼'指北 — 蟒营™ 怂怼录](https://blog.101.camp/dm/200220-dm30-imho-how2work-dark/)
- [拙见/ 科学'摸鱼'指北+1 — 蟒营™ 怂怼录](https://blog.101.camp/dm/200221-dm31-imho-how2work-deep/)
- [拙见/ 科学'摸鱼'指北+2 — 蟒营™ 怂怼录](https://blog.101.camp/dm/200221-dm32-imho-how2work-debug/)

当然, 都是玩笑...

其实, 时间是什么? 时间不过是有印象的行为记忆间隔 ;-)

人生最重要的是开心, 只作自己想作的事儿是最幸福的,
可惜多数人根本不知道自己想作什么, 
只知道社会/家庭/组织/弟兄们/...要自己作什么...

所以, 有效规划时间这事儿, 从根儿上还是确立一个 [SMART原则 - MBA智库百科](https://wiki.mbalib.com/wiki/SMART%E5%8E%9F%E5%88%99) 任务而已.
